Makefile

```diff
@@ -158,7 +158,7 @@ nuttx_% posix_% qurt_%:
 # Abbreviated config targets.
 
 # nuttx_ is left off by default.
-tap-% asc-% px4fmu-% px4-% mindpx-% crazyflie_default:
+tap-% asc-% px4fmu-% px4-% mindpx-% crazyflie_%:
```

Comment:
The underscore here looks sort of wrong - are we sure crazyflie is actually using one?